onnx/defs/math/defs.cc

```diff
@@ -1473,6 +1473,65 @@ ONNX_OPERATOR_SET_SCHEMA(
 
           matmulShapeInference(ctx, 0, 1);
         }));
+static const char* CumSum_ver11_doc = R"DOC(
+Performs cumulative sum of the input elements along the given axis.
+By default, it will do the sum inclusively meaning the first element is copied as is.
+Through an `exclusive` attribute, this behavior can change to exclude the first element.
+It can also perform summation in the opposite direction of the axis. For that, set `reverse` attribute to 1.
+
+Example:
+```
+input_x = [1, 2, 3]
+axis=0
+output = [1, 3, 6]
+exclusive=1
+output = [0, 1, 3]
+exclusive=0
+reverse=1
+output = [6, 3, 1]
+exclusive=1
+reverse=1
+output = [3, 1, 0]
+```
+ )DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+      CumSum,
+      11,
+    OpSchema()
+            .SetDoc(CumSum_ver11_doc)
+      .Attr(
+          "exclusive",
+          "If set to 1 will return exclusive sum in which the top element is not included.",
```

Comment:
How about "If set to 1, the j-th output element is the sum of the first (j-1) elements. Otherwise, it is the sum of the first j elements."?

Reply:
I'll add your line as well if that makes it more clear

Reply:
Yes, that would be helpful (IMO).